docs/pages/reference/js/getFrame.mdx

```diff
@@ -1,25 +1,37 @@
 # getFrame
 
-Returns a `Frame` object, extracting the frame metadata from the given htmlString, or returning null if the frame is invalid. Includes tags from the [Open Frames specification](https://github.com/open-frames/standard).
+Returns an object with `farcaster` and `openframes` keys, each containing an object with `frame` property extracting the frame metadata from the given htmlString, or containing `reports` property if the frame is invalid. Includes tags from the [Open Frames specification](https://github.com/open-frames/standard).
```

Comment:
I think you are right. That was really the part I struggled the most with. So by default we should probably keep the `farcaster` since that's how the lib works atm right?

Really appreciate your feedback here since that gave me new ideas. I'm going to implement your suggestion!

Reply:
Yeah i think that's the right thing to do for now. Also just a correction, the wrapper `getFrame` should maybe accept an array of client protocol IDs in case the client application has signers for multiple protocols or maybe wants to support multiple versions of the same protocol

Reply:
I think it will be better to do that in other PR when there will be a need for that since I don't want to to block you.